Whiskey/Tasks/GetPowerShellModule.ps1

```diff
@@ -50,7 +50,6 @@ function Get-WhiskeyPowerShellModule
                                               -Version $Version `
                                               -BuildRoot $TaskContext.BuildRoot `
                                               -AllowPrerelease:$AllowPrerelease `
-                                              -Path $fullPath `

```

Comment:
Are we removing the ability to specify a path for where the PowerShell modules will be downloaded to?

Reply:
No. We still use it when we install. We just don't need it when we find the version to install. Not sure why I ever passed the path to `Resolve-WhiskeyPowerShellModule`.

Reply:
👍